cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/log/MockLogReader.java

```diff
@@ -52,49 +53,51 @@
 
     // Add log lines for app testApp1, flow testFlow1
     for (int i = 0; i < MAX; ++i) {
-      logMap.put(defaultNamespacedLogBaseDir + "/testApp1/flow-testFlow1", new LogLine(i, "testFlow1<img>-" + i));
+      logMap.put(defaultNamespacedLogBaseDir + "/testApp1/flow-testFlow1",
+                 new LogLine(new LogOffset(i, i), "testFlow1<img>-" + i));
     }
 
     // Add log lines for app testApp1, service testService1
     for (int i = 0; i < MAX; ++i) {
       logMap.put(defaultNamespacedLogBaseDir + "/testApp4/userservice-testService1",
-                 new LogLine(i, "testService1<img>-" + i));
+                 new LogLine(new LogOffset(i, i), "testService1<img>-" + i));
     }
 
     // Add log lines for app testApp2, procedure testProcedure1
     for (int i = 0; i < MAX; ++i) {
       logMap.put(defaultNamespacedLogBaseDir + "/testApp2/procedure-testProcedure1",
-                 new LogLine(i, "testProcedure1<img>-" + i));
```

Comment:
We already have a test for service. If procedure is removed I can remove this test, else we can remove it when procedure gets removed.